Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM golang:1.8 AS build
+ARG GO_VERSION=1.17
```

Comment:
Perhaps we should adjust the versions we test in CI accordingly

(I see the `go.mod` still lists `go 1.11`. that's _probably_ fine, but we don't test that version anymore, so it _could_ be broken without knowing)